cfgov/unprocessed/css/pages/ask.less

```diff
@@ -194,16 +199,38 @@
         }
     } ); 
 
-    .about-us-text,
-    .disclaimer-text {
-        a {
-            font-weight: 500;
+    .o-sidebar-content {
+        .about-us-text,
+        .disclaimer-text {
+            a {
+                font-weight: 500;
+            }
+        }
+
+        .about-us-text {
+            margin-bottom: unit( @grid_gutter-width * 1.5 / @base-font-size-px, em );
+        }
+
+        .about-us-text + .block {
+            margin-top: unit( @grid_gutter-width * 2 / @base-font-size-px, em ) !important;
+        }
+
+        .about-us-text + .disclaimer-text {
+            border-top: 1px solid @block__border;
+            margin-top: unit( @grid_gutter-width * 1.5 / @base-font-size-px, em );
+            padding-top: unit( ( @grid_gutter-width * 1.5 ) / @base-font-size-px, em );
```

Comment:
VERY TRIVIAL: extra parens in this line versus the line above, but it's the same setting.

Reply:
Thanks for catching that, @anselmbradford -- fixed in 027006168e34fee9ae276dcca21092a4618b7ace